### DISCUSSION TITLE
HIP-30 - CAIP Identifiers for the Hedera Network

### DISCUSSION THREAD

**Discussion by shemnon** [hiero-ledger/hiero-improvement-proposals#169] [Edits]
Chain Agnostic Improvement Proposals (CAIP) define chain interop standards.  This HIP describes the set of identifiers Hedera will use and will also be defined in CAIP as well via 4 CAIPs.

**Comment by rocketmay** [comment]
@shemnon Hello, my team and I are interested in pushing this through. Have you written the CAIP or submitted the PR to the CAIP github?

**Reply by shemnon** [reply]
I've submitted CAIP-75 through CAIP-78 to cover the blockchain, account, token, and NFT namespace.  Now I need their editors to approve and merge the drafts.

**Reply by shemnon** [reply]
https://github.com/ChainAgnostic/CAIPs/pull/75 - blockchain
https://github.com/ChainAgnostic/CAIPs/pull/76 - account
https://github.com/ChainAgnostic/CAIPs/pull/77 - token
https://github.com/ChainAgnostic/CAIPs/pull/78 - NFT

**Reply by rocketmay** [reply]
Thank you for the quick turnaround!
